src/lib.rs

```diff
@@ -161,13 +137,19 @@ pub fn encode_composition(
             &virt,
             "`spin-test-virt`",
         ),
+        ("wasi:cli/environment@0.2.0", &virt, "`spin-test-virt`"),
```

Comment:
I don't think we want to do this. This will block tests themselves from using environment variables, and I don't think that's what we want. I think we want to allow tests to use the host environment - only the app should have its environment virtualized.